app/managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self.connection.execute(
+            f"INSERT INTO {self.table_name} "
+            f"(first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
```

Comment:
Using f-strings for SQL queries can lead to SQL injection vulnerabilities. It's safer to use parameterized queries, which you are already doing with placeholders (?, ?). Ensure that the table name is not user-controlled to avoid SQL injection through the table name.